### DISCUSSION TITLE
"View Open Tabs" keybind

### DISCUSSION THREAD

**Discussion by hossamkhero** [ghostty-org/ghostty#3886] [Q&A] [answered]
a keybind to perform the same action as the "View Open Tabs" button in the titlebar on gtk, for cases when someone want to perform this action but has the titlebar disabled, and the keybind can look something like this

keybind = ctrl+shift+v=view_open_tabs

**Comment by mitchellh** [comment, marked as answer]
`toggle_tab_overview` exists

**Reply by jgarte** [reply]
Would it be possible to just provide an opinionated keybinding for this out of the box not unlike gnome does with the `super` key?
